be/src/models/user.model.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, Column, PrimaryGeneratedColumn, Unique, BeforeInsert, BeforeUpdate } from 'typeorm';
+import { ObjectType, Field } from '@nestjs/graphql';
+
+
+@ObjectType()
+@Entity('users')
+export class UserModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Field()
+  @Column({ length: 255, nullable: false })
+  name: string;
+  @Column({ length: 255, nullable: false})
+  public password: string;
```

Comment:
yes i know at the moment it keeps plaintext password. did not figure out how to get mutations working so it would hash password on insert or edit

Reply:
I recommend to add password encryption, using bcrypt will do good job. But for now it will suffice.